manifests/init.pp

```diff
@@ -104,8 +107,14 @@
   Optional[String[1]] $config_template_v6       = undef,
   Boolean $manage_repo                          = false,
   Optional[String[1]] $config_content_v4        = undef,
-  Optional[String[1]] $config_content_v6        = undef
+  Optional[String[1]] $config_content_v6        = undef,
+  Boolean $purge_unmanaged_snippets             = true,
```

Comment:
Just a note that we should do a major bump on this as it's a significant change to behaviour

Reply:
changed it to false so it's not a breaking change